Cargo.lock

```diff
@@ -235,6 +272,16 @@ version = "0.11.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "ab5ef0d4909ef3724cc8cce6ccc8572c5c817592e9285f5464f8e86f8bd3726e"
 
+[[package]]
+name = "iana-time-zone"
+version = "0.1.33"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "df1b4d6691512a74af43bb6de0b7f546fbe70bfa3bac84da8454a00121e63201"
+dependencies = [
+ "core-foundation",
+ "winapi",
```

Comment:
I'm 99% certain this crate is required because of `local()`. e.g. if we don't use `local()`, I think you get `Time` in wasm, but just without local timezone discovery.

I'll add a feature (`tzrs-local`?) around the function for getting the local timezone. Opinion: Should `local()` in an environment panic, or just return utc? :thinking:

Reply:
I've added `tzrs-local` as a feature flag. It returns GMT if disabled. Documented the behavior and added to feature flags docs.